content/software-security/cves/cve-why-care.md

```diff
@@ -0,0 +1,58 @@
+
+Addressing the vulnerabilities present in your software helps secure the systems you support, use, and maintain. In this article, you will explore why you should care about vulnerabilities as a software developer, and learn about federal regulations that draw importance to CVE management (such as FedRAMP).
+
+
+## As a Developer
+
+Discovering and mitigating software vulnerabilities is a difficult – but necessary – task for developers to tackle. Development teams are ultimately the authorities which are able to remediate vulnerabilities at the source. Depending on the severity of a vulnerability, its exploitation could significantly impact the integrity of the software product, as well as the trust, safety, and operations of the consumers who rely on it.
+
+Vulnerability *triage* is an important step developers must take to work toward reducing risks posed by vulnerabilities in their software. Using [*vulnerability scanners*](https://owasp.org/www-community/Vulnerability_Scanning_Tools) can give developers information about the number, type, and severity of CVEs present in their work. With this data, developers can prioritize critical CVEs first, therefore ensuring that major security concerns are addressed first.
+
+
+## Federal Regulations
+
+With notable software supply chain security attacks occurring in recent years (such as the [SolarWinds attack in 2020](/software-security/glossary/#solarwinds-hack)), the U.S. federal government has increased efforts to improve U.S. cybersecurity. These initiatives aim to strengthen software supply chain security by promoting safe development habits, such as frequent vulnerability scanning.
+
+
+### FedRAMP
+
+The [Federal Risk and Authorization Management Program](https://www.fedramp.gov/program-basics/) (FedRAMP) is a security framework that must be adopted by cloud service providers (CSPs) before they service U.S. federal government agencies. The framework aims to promote the use of cloud services across agencies by standardizing security authorization practices. Based on the service being offered, FedRAMP sorts cloud services into low, moderate, and high impact levels, with increased security expectations for each level.
+
+In order to achieve FedRAMP authorization, certain requirements need to be met as follows: 
+
+- Container images used by CSPs must be *hardened* according to benchmarks laid out in the [NIST SP 800-70](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-70r4.pdf). Chainguard Images offers [hardened, minimal base images](https://www.chainguard.dev/chainguard-images) designed to help meet FedRAMP compliance requirements.
```

Comment:
You can use the royal "we" to talk about Chainguard here if that seems more natural. Like "We developed one solution, Chainguard Images..."

```suggestion
- Container images used by CSPs must be *hardened* according to benchmarks laid out in the [NIST SP 800-70](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-70r4.pdf). One solution here is Chainguard Images, which offers [hardened, minimal base images](https://www.chainguard.dev/chainguard-images) designed to help meet FedRAMP compliance requirements.
```